### PR TITLE
reworked homepage to be responsive

### PR DESCRIPTION
**Pull request by insectengine**
Reworked the home page and footer to be responsive with breakpoints for desktop, tablet and mobile.

### CODE REVIEW

File: themes/ansible-community/sass/redhat-footer.scss [hunk 1]
```diff
@@ -56,7 +61,39 @@
   list-style-type: none;
   padding-left: unset;
 
-  li:not(:first-child) {
-    margin-top: 1.5rem;
+  li {
+  display: inline;
+  margin-right: 1rem;
+  white-space; nowrap;
```
Comment (GregSutcliffe) [inline themes/ansible-community/sass/redhat-footer.scss:67]: should be a : not a ;

File: templates/homepage-banner-band.tmpl [hunk 2]
```diff
@@ -34,8 +34,7 @@
       <div class="banner-welcome-image">
         <img src="/images/bullito_welcome.svg"
              alt="{{ homepage.banner.welcome_alt }}"
-             width="350"
-             height="300" />
+             />
```
Comment (GregSutcliffe) [inline templates/homepage-banner-band.tmpl:37]: This is failing the linter for specifying height/width, is there a reason to delete it? (I.e does our linter need changes instead?)

File: templates/homepage-events-band.tmpl [hunk 3]
```diff
@@ -18,19 +18,12 @@
       {% for key, item in homepage.events.events_list.items() %}
         <div class="events-box">
           <div class="events-content">
-            <a href="{{ item.link }}"
+<a href="{{ item.link }}"
                title="Link to the event in the Discourse calendar.">
               <img src="/images/{{ item.image }}"
                    alt="{{ homepage.events.event_alt }}"
-                   width="720"
```
Comment (GregSutcliffe) [inline templates/homepage-events-band.tmpl:25]: Same comment, this fails the height/width lint

**Review by GregSutcliffe — CHANGES_REQUESTED** [review]
Thanks @insectengine! The linter is failing a couple of things, and the templates also have some suggested restyling - do you want me to add a commit for that or would you rather address it yourself?

**Comment by oraNod** [comment]
Thanks for the awesome work here @insectengine I've pulled the changes into https://github.com/ansible-community/community-website/issues/47

**Pull request closed without merging**